### PR TITLE
[Blenderbot] Fix Links

### LINKED ISSUE #9527: [BlenderbotSmallTokenizer] Cannot download tokenizer
When running: 

```python
from transformers import BlenderbotSmallTokenizer
tok = BlenderbotSmallTokenizer.from_pretrained("facebook/blenderbot_small-90M")
```

the command fails with the error

```~/python_bin/transformers/tokenization_utils_base.py in _from_pretrained(cls, resolved_vocab_files, pretrained_model_name_or_path, init_configuration, *init_inputs, **kwargs)
   1894         # Instantiate tokenizer.
   1895         try:
-> 1896             tokenizer = cls(*init_inputs, **init_kwargs)
   1897         except OSError:
   1898             raise OSError(

~/python_bin/transformers/models/blenderbot_small/tokenization_blenderbot_small.py in __init__(self, vocab_file, merges_file, bos_token, eos_token, unk_token, pad_token, **kwargs)
    107 
    108         with open(vocab_file, encoding="utf-8") as vocab_handle:
--> 109             self.encoder = json.load(vocab_handle)
    110         self.decoder = {v: k for k, v in self.encoder.items()}
    111         with open(merges_file, encoding="utf-8") as merges_handle:

/usr/lib/python3.7/json/__init__.py in load(fp, cls, object_hook, parse_float, parse_int, parse_constant, object_pairs_hook, **kw)
    294         cls=cls, object_hook=object_hook,
    295         parse_float=parse_float, parse_int=parse_int,
--> 296         parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
    297 
    298 

/usr/lib/python3.7/json/__init__.py in loads(s, encoding, cls, object_hook, parse_float, parse_int, parse_constant, object_pairs_hook, **kw)
    346             parse_int is None and parse_float is None and
    347             parse_constant is None and object_pairs_hook is None and not kw):
--> 348         return _default_decoder.decode(s)
    349     if cls is None:
    350         cls = JSONDecoder

/usr/lib/python3.7/json/decoder.py in decode(self, s, _w)
    335 
    336         """
--> 337         obj, end = self.raw_decode(s, idx=_w(s, 0).end())
    338         end = _w(s, end).end()
    339         if end != len(s):

/usr/lib/python3.7/json/decoder.py in raw_decode(self, s, idx)
    353             obj, end = self.scan_once(s, idx)
    354         except StopIteration as err:
--> 355             raise JSONDecodeError("Expecting value", s, err.value) from None
    356         return obj, end

JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

This is strange since `"facebook/blenderbot_small-90M"` is just a copy of `"facebook/blenderbot-90M"` which works:

```python
from transformers import BlenderbotSmallTokenizer
tok = BlenderbotSmallTokenizer.from_pretrained("facebook/blenderbot-90M")
```

### PR DESCRIPTION
**Pull request by patrickvonplaten**
# What does this PR do?

Fixes #9527

Credit goes to @LysandreJik  for finding the fix.

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors which may be interested in your PR.

### CODE REVIEW

File: src/transformers/models/blenderbot_small/tokenization_blenderbot_small.py [hunk 1]
```diff
@@ -82,13 +82,13 @@ class BlenderbotSmallTokenizer(PreTrainedTokenizer):
     }
     pretrained_vocab_files_map = {
         "vocab_file": {
-            "facebook/blenderbot_small-90M": "https://huggingface.co/facebook/blenderbot_small-90M/blob/main/vocab.json"
+            "facebook/blenderbot_small-90M": "https://huggingface.co/facebook/blenderbot_small-90M/resolve/main/vocab.json"
```
Comment (patrickvonplaten) [inline src/transformers/models/blenderbot_small/tokenization_blenderbot_small.py:85]: That's actually the thing I messed up

**Review by LysandreJik — APPROVED** [review]
LGTM!

**Comment by julien-c** [comment]
Ok now's really the time to remove those hard-coded links once and for all, i think

**Comment by LysandreJik** [comment]
Indeed! It's on my todo for the week.

**Pull request merged**